backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/webservice/schema/ApiTokenService.kt

```diff
@@ -32,13 +32,11 @@ class ApiTokenService {
         val admin = transaction { AdministratorEntity.findById(jwtPayload.adminId) }
         admin?.takeIf { Authorizer.mayAddApiTokensInProject(it) } ?: throw ForbiddenException()
 
-        val random = SecureRandom()
         val bytes = ByteArray(getByteArrayLength())
-        random.nextBytes(bytes)
-        val token = Base64.getEncoder()
-            .encodeToString(bytes)
+        SecureRandom().nextBytes(bytes)
+        val token = Base64.getEncoder().encodeToString(bytes)
 
-        val encryptedToken = BCrypt.withDefaults().hash(11, token.toCharArray())
+        val encryptedToken = PasswordCrypto.hashWithSHA256(token.toByteArray())
```

Comment:
It's not encrypted, it's just hashed, right?
```suggestion
        val tokenHash = PasswordCrypto.hashWithSHA256(token.toByteArray())
```